stdlib/public/core/HashedCollections.swift.gyb

```diff
@@ -3843,6 +3858,63 @@ internal enum _Variant${Self}Storage<${TypeParametersDecl}> : _HashStorage {
     }
   }
 
+  internal mutating func nativeInsert(
+    _ value: Value, forKey key: Key
+  ) -> (inserted: Bool, memberAfterInsert: Value) {
+    var (i, found) = asNative._find(key, startBucket: asNative._bucket(key))
+
+    if found {
+%if Self == 'Set':
+      return (inserted: false, memberAfterInsert: asNative.key(at: i.offset))
+%elif Self == 'Dictionary':
+      return (inserted: false, memberAfterInsert: asNative.value(at: i.offset))
+%end
+    }
+
+    let minCapacity = NativeStorage.minimumCapacity(
+                        minimumCount: asNative.count + 1,
+                        maxLoadFactorInverse: asNative.maxLoadFactorInverse)
```

Comment:
Please format this with 2 spaces on continuation line, the library style is different from LLVM style:

```
+let minCapacity = NativeStorage.minimumCapacity(
+  minimumCount: asNative.count + 1,
+  maxLoadFactorInverse: asNative.maxLoadFactorInverse)
```